### PR TITLE
Correct steam usage calculation for geared locomotives

### PR DESCRIPTION
**Pull request by peternewell**
https://bugs.launchpad.net/or/+bug/1899443

### CODE REVIEW

File: Source/Orts.Simulation/Simulation/RollingStocks/MSTSSteamLocomotive.cs [hunk 1]
```diff
@@ -4383,7 +4383,10 @@ private void UpdateCylinders(float elapsedClockSeconds, float throttle, float cu
                     CylinderAdmissionSteamWeightLbs = 0.0f;
                 }
                 RawCylinderSteamWeightLbs = CylinderReleaseSteamWeightLbs - CylinderAdmissionSteamWeightLbs;
-                RawCalculatedCylinderSteamUsageLBpS = NumCylinders * DrvWheelRevRpS * CylStrokesPerCycle * (RawCylinderSteamWeightLbs);
+
+                // Calculate steam usage based upon how many piston strokes happen ffor each revolution of the wheels. 
+                // Geared locomotives will have to take into account gearing ratio. 
+                RawCalculatedCylinderSteamUsageLBpS = NumCylinders * DrvWheelRevRpS * MotiveForceGearRatio * CylStrokesPerCycle * (RawCylinderSteamWeightLbs);
```
Comment (YoRyan) [inline Source/Orts.Simulation/Simulation/RollingStocks/MSTSSteamLocomotive.cs:4389]:
```suggestion
                RawCalculatedCylinderSteamUsageLBpS = NumCylinders * DrvWheelRevRpS * MotiveForceGearRatio * CylStrokesPerCycle * RawCylinderSteamWeightLbs;
```
Suuuuuper minor thing: Let's drop the extraneous parantheses while we're here. 🙂

**Review by YoRyan — CHANGES_REQUESTED** [review]

File: Source/Orts.Simulation/Simulation/RollingStocks/MSTSSteamLocomotive.cs (hunk 1 above)
Comment (peternewell) [reply Source/Orts.Simulation/Simulation/RollingStocks/MSTSSteamLocomotive.cs]: Done

**Review by YoRyan — APPROVED** [review]
Thanks, it looks good to me. Let's give it a few more days in the unstable branch before releasing it to testing.

**Review by YoRyan — APPROVED** [review]

**Comment by YoRyan** [comment]
Possible regression? http://www.elvastower.com/forums/index.php?/topic/34561-problems-starting-a-train-from-a-stop/

**Comment by peternewell** [comment]
> Possible regression? 

Not at this stage.

Firstly the changes in this PR should only impact geared steam locomotives, and not diesels.

Secondly the issue described needs some more definition. I will have a look at it when I get some time.

**Pull request merged**